SAMLPortal/Models/GlobalSettings.cs

```diff
@@ -20,29 +22,53 @@ public static class GlobalSettings
 		/// </summary>
 		public static void InitSettingsFromEnvironment()
 		{
-			DotNetEnv.Env.Load("./.env");
-			IDictionary environment = Environment.GetEnvironmentVariables();
+			FillSettingsArray();
 
-			if (Environment.GetEnvironmentVariable("SP_FORCE_ENV_REPOPUL") == "1")
+			string configurationPath = GlobalSettings.Get("CONFIG_PATH");
+			if (configurationPath != null)
 			{
-				// Re-fill databse with environment configuration
-				foreach (var key in environment.Keys)
+				if (Directory.Exists(configurationPath))
 				{
-					if (key.ToString().StartsWith("SP_") && !key.ToString().StartsWith("SP_MYSQL"))
+					// Ensure trailing slash is present
+					configurationPath = configurationPath.TrimEnd('/') + '/';
+
+					var configPath = configurationPath + "config.env";
+					if (!File.Exists(configPath))
 					{
-						string shortenedKey = key.ToString().Remove(0, 3);
-						_appSettings.Add(shortenedKey, environment[key].ToString());
+						using (FileStream fs = File.Create(configPath)) { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)